.cirrus.yml

```diff
@@ -491,11 +491,48 @@ task:
     - git clean -xfd
     - git fetch origin
     - git fetch origin master # To set FETCH_HEAD for "git merge-base" to work
+  pub_cache:
```

Comment:
I don't know that this is necessary.

Reply:
We should remove all the _cache from these -they're actually slower on the macs